hubmap/frontend/src/components/Experiments.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import grey from '@material-ui/core/colors/grey';
+import PropTypes from 'prop-types';
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+
+const StyledTableCell = withStyles(theme => ({
+  head: {
+    backgroundColor: grey[800],
+    color: theme.palette.common.white,
+  },
+  body: {
+    fontSize: 14,
+  },
+}))(TableCell);
+
+const StyledTableRow = withStyles(theme => ({
+  root: {
+    '&:nth-of-type(odd)': {
+      backgroundColor: grey[300],
+    },
+  },
+}))(TableRow);
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: '100%',
+    marginTop: theme.spacing(3),
+    overflowX: 'auto',
+  },
+  table: {
+    minWidth: 700,
+  },
+}));
+
+class Experiments extends React.Component {
+  state = {
+    experiments: [],
+    searchString: ''
+  }
+
+  getExperimentsTable(experiments) {
+    return (<Paper className={useStyles.root}>
+      <Table className={useStyles.table} title="Studies from HuBMAP Consortium">
+        <TableHead>
+          <TableRow backgroundColor={grey[300]}>
+            <StyledTableCell align="right">Id</StyledTableCell>
+            <StyledTableCell align="right">Study type</StyledTableCell>
+            <StyledTableCell align="right">Institution</StyledTableCell>
+            <StyledTableCell align="right">Data type</StyledTableCell>
+            <StyledTableCell align="right">Tissue</StyledTableCell>
+            <StyledTableCell align="right">Uploaded on</StyledTableCell>
+            <StyledTableCell align="right">Genes </StyledTableCell>
+            <StyledTableCell align="right">Proteins</StyledTableCell>
+            <StyledTableCell align="right"># Cells</StyledTableCell>
+            <StyledTableCell align="right"># Unique barcode </StyledTableCell>
+            <StyledTableCell align="right"># Reads </StyledTableCell>
+            <StyledTableCell align="right"># Reads Aligned </StyledTableCell>
+            <StyledTableCell align="right"># Images </StyledTableCell>
```

Comment:
Would it be more clear to just use a map?
```jsx
const heads = ['Id', 'Study type', 'Institution', /* etc */ ].map(
  label => <StyledTableCell align="right">{label}</StyledTableCell>
);
...
<TableRow backgroundColor={grey[300]}>{heads}</TableRow>
```

Reply:
I see, I will take a look into these later sometime this week once auth is complete. Appreciate your comments.